setup.cfg

```diff
@@ -1,5 +1,8 @@
 [flake8]
 max-line-length = 100
+ignore =
+    # ...manually surrounded by quotes, consider using !r
+    B028
```

Comment:
suppresses this new check from `flake8-bugbear` that I disagree with.

```text
./src/pydistcheck/config.py:71:30: B028 'name' is manually surrounded by quotes, consider using the `!r` conversion flag.
./src/pydistcheck/checks.py:129:21: B028 'ascii_converted_str' is manually surrounded by quotes, consider using the `!r` conversion flag.
./src/pydistcheck/checks.py:145:21: B028 'file_path' is manually surrounded by quotes, consider using the `!r` conversion flag.
./src/pydistcheck/checks.py:167:27: B028 'file_path' is manually surrounded by quotes, consider using the `!r` conversion flag.
./src/pydistcheck/checks.py:177:27: B028 'directory_path' is manually surrounded by quotes, consider using the `!r` conversion flag.
./src/pydistcheck/cli.py:173:15: B028 'filepath' is manually surrounded by quotes, consider using the `!r` conversion flag.
./tests/test_docs.py:44:12: B028 'check' is manually surrounded by quotes, consider using the `!r` conversion flag.
./tests/test_cli.py:21:9: B028 'pattern' is manually surrounded by quotes, consider using the `!r` conversion flag.
./bin/get-release-files.py:15:7: B028 'PACKAGE_NAME' is manually surrounded by quotes, consider using the `!r` conversion flag.
./bin/get-release-files.py:49:11: B028 'sample_release.filename' is manually surrounded by quotes, consider using the `!r` conversion flag.
./bin/get-release-files.py:55:7: B028 'OUTPUT_DIR' is manually surrounded by quotes, consider using the `!r` conversion flag.
```

Reply:
just kidding, they already reverted this: #333

removed this

Reply:
in https://github.com/jameslamb/pydistcheck/pull/147/commits/b92bf7274437714b971dd103a4fae5ae01b349d1